### ISSUE TITLE
Resizing of panes doesn't work since update to Gnome 44.1

### ISSUE THREAD

**Issue by speightashley** [material-shell/material-shell#963]
**Describe the bug**
Maximise works fine. The issue comes when we go into a tiling mode that shares real estate. I code on the left with a bigger pane than on the right which is generally a browser to see what I build in real time. When hovering over the split line, when I drag to resize, it no longer does it and just stays static. It tries to do it based on the content resizing in the windows but it keeps the same dimensions for VSCode and Firefox. 

**Expected behavior**
Window on the left gets bigger when dragging to the right / window on the right gets smaller. 

**To Reproduce**
- Occurs on:
  - [ ] X.Org
  - [ X ] Wayland
- GNOME Shell version: 44.1
- Distro: Manjaro

**Comment by ademoverflow** [comment]
Same thing since the update .. very annoying :(

**Comment by Luctins** [comment]
Same thing here with Fedora 38, Gnome Shell 44.2 @ 89f460b (current HEAD, installed from source).

I would advise in the meantime to use the float layout with pop-shell as the tiler (so you still have persistence).

Sample video:
![demo](https://github.com/material-shell/material-shell/assets/4743486/d5713828-25f7-497b-922c-4925e42f7372)

**Comment by Luctins** [comment]
Update: It seems that only 'integer multiple' applications are resizeable, like terminals (that can only be a integer multiple of characters wide).

**Comment by ademoverflow** [comment]
Any update on this ?

**Comment by PapyElGringo** [comment]
Yeah sorry for the delay. I'm aware of the issue and it's because Gnome removed an API from Mutter that allowed us to listen to window events and allowed us to take priority on it to make our resize or Drag and drop features.

I already talked to them about it and they said it wasn't a valid use case so I can't simply fix it. I need to find an alternative strategy and make a rework of those features until another thing break...

Since it's still planned to work on our own WM is the short future I'm not sure I'll dig into it for the extension

**Comment by donalffons** [comment]
Thanks @PapyElGringo for your great work on this project.

> Since it's still planned to work on our own WM is the short future I'm not sure I'll dig into it for the extension
Could you clarify what this means? Are you working on a "Material Shell" Window Manager as an alternative to Gnome? Or what is "the short future"?

**Comment by Mempler** [comment]
> Since it's still planned to work on our own WM is the short future I'm not sure I'll dig into it for the extension

will it be X11 based or will it be Wayland based ?
I would love to see the future of this project.

**Comment by ademoverflow** [comment]
Any update on this issue ?

**Comment by Luctins** [comment]
I still use my somewhat crappy workaround of Increasing the margin between
windows and dragging that (you need to build the extension from source).

I don't remember exactly if it is gone on gnome 45, but I can test later.
